tests/test_basic_s3.py

```diff
@@ -174,14 +174,28 @@ async def test_result_key_iters(s3_client, bucket_name, create_object):
     assert 'CommonPrefixes' in response
 
 
-async def test_can_get_and_put_object(s3_client, create_object, bucket_name):
-    await create_object('foobarbaz', body='body contents')
-    resp = await s3_client.get_object(Bucket=bucket_name, Key='foobarbaz')
+async def test_can_get_and_put_object(
+    s3_client, create_object, bucket_name: str
```

Comment:
Why that single type annotation?

Reply:
because s3_client is a dynamic object and create_object is a function.  Just stepping into the right direction